src/test/javascript/spec/component/exam/manage/student-exams/user-import-dialog.component.spec.ts

```diff
@@ -174,36 +179,35 @@ describe('UsersImportButtonComponent', () => {
         const studentsNotFound: StudentDTO[] = [{ registrationNumber: '3', firstName: 'Some', lastName: 'Dude', login: 'login3' }];
 
         const fakeResponse = { body: studentsNotFound } as HttpResponse<StudentDTO[]>;
-        sinon.replace(examManagementService, 'addStudentsToExam', sinon.fake.returns(of(fakeResponse)));
+        jest.spyOn(examManagementService, 'addStudentsToExam').mockReturnValue(of(fakeResponse));
 
         component.usersToImport = studentsToImport;
 
         fixture.detectChanges();
 
-        expect(component.hasImported).to.be.false;
-        expect(component.isSubmitDisabled).to.be.false;
+        expect(component.hasImported).toBeFalse;
+        expect(component.isSubmitDisabled).toBeFalse;
```

Comment:
```suggestion
        expect(component.hasImported).toBe(false);
        expect(component.isSubmitDisabled).toBe(false);
```
No idea why this even compiles but anyway